api/helpers/game-states/moves/stalemate-accept/execute.js

```diff
@@ -0,0 +1,52 @@
+const GamePhase = require('../../../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Accept offer for Stalemate',
+
+  description: 'Accept opponent\'s stalemate offer',
```

Comment:
TODO: write an issue to standardize string use when `'` characters are in the string. I think `\'` is the cleanest, compared with using `""` or backtick-strings just for '

Reply:
https://github.com/cuttle-cards/cuttle/issues/1181